.storybook/themes/news.scss

```diff
@@ -456,6 +459,252 @@
+					font-style: italic,
+					gap: var(--global-spacing-6),
+					padding: var(--global-spacing-4) 0 var(--global-spacing-4) var(--global-spacing-5),
+					text-align: left,
+					components: (
+						paragraph: (
+							font-size: var(--global-font-size-5),
+						),
+					),
+				),
+				article-body-children: (
+					margin-bottom: var(--global-spacing-5),
+				),
+				article-body-citation: (
+					font-family: var(--font-family-primary),
+					font-style: normal,
+					font-size: 16px,
+				),
+				article-body-correction: (
+					box-shadow: inset 0 -1px 0 0 var(--border-color),
+					font-style: italic,
+					padding-bottom: var(--global-spacing-6),
+					components: (
+						heading: (
+							font-size: var(--global-font-size-3),
+							text-transform: uppercase,
+						),
+						paragraph: (
+							color: var(--global-neutral-6),
+						),
+					),
+				),
+				article-body-divider: (
+					background-color: var(--border-color),
+					height: 1px,
+				),
+				article-body-gallery: (
+					components: (
+						icon: (
+							fill: var(--global-white),
+							height: var(--global-spacing-8),
+							width: var(--global-spacing-8),
+						),
+						carousel-button: (
+							background-color: transparent,
+							box-shadow: none,
+						),
+						carousel-expand-autoplay-container: (
+							gap: var(--global-spacing-4),
+						),
+						carousel-button-enter-full-screen: (
+							background-color: transparent,
+							border: none,
+							box-shadow: none,
+							color: var(--text-color),
+							font-size: var(--global-font-size-3),
+							height: auto,
+							width: auto,
+						),
+						carousel-button-toggle-auto-play: (
+							background-color: transparent,
+							box-shadow: none,
+							color: var(--text-color),
+							font-size: var(--global-font-size-3),
+							height: auto,
+							width: auto,
+						),
+						carousel-image-counter: (
+							font-size: var(--global-font-size-3),
+						),
+						media-item: (
+							background-color: var(--global-neutral-9),
+							gap: 0,
+							justify-content: center,
+							width: 100%,
+						),
+						media-item-fig-caption: (
+							background-color: var(--global-white),
+							padding-top: var(--global-spacing-2),
+						),
+					),
+				),
+				article-body-gallery-additional-next: (
+					border: none,
+					display: none,
+					height: var(--global-spacing-6),
+					width: var(--global-spacing-6),
+					components: (
+						icon: (
+							fill: var(--text-color),
+							height: var(--global-spacing-4),
+							width: var(--global-spacing-4),
+						),
+					),
+				),
+				article-body-gallery-additional-previous: (
+					border: none,
+					display: none,
+					height: var(--global-spacing-6),
+					width: var(--global-spacing-6),
+					components: (
+						icon: (
+							fill: var(--text-color),
+							height: var(--global-spacing-4),
+							width: var(--global-spacing-4),
+						),
+					),
+				),
+				article-body-image-wrapper: (
```

Comment:
can the options align-items here be hard-coded with props and a stack? or was the intention to allow clients to override this?

Reply:
The element is not a stack this is a `div` in the code - and for our styling needs we need to use this CSS properties

Reply:
ok